script/test_prism.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euxo pipefail
+
+rm -rf tmp/
+source ./script/functions.sh
+
+export RUBYFMT_PRISM=1
+cargo build --release
+
+uname -a
+
+# TODO(reese): uncomment these as we build out Prism support
+# ./script/tests/test_string_literals.sh
+# ./script/tests/test_array_literals.sh
+# ./script/tests/test_methods.sh
+# ./script/tests/test_cli_interface.sh
+# ./script/tests/test_c_main.sh
+# ./script/tests/test_error_handling.sh
+# ./script/tests/test_fixtures.sh
+FIXTURE_NAME=numbers ./script/tests/test_fixtures.sh
+# ./script/tests/test_formatting_locks.sh
```

Comment:
Long-term, I think we will be much better served by porting this into proper `#[test]` blocks, for parallelism and otherwise.  It's not obvious to me that we can do parallel tests in a single Rust process with things written as they are today, though.

Reply:
Yeah I may do that after this, I'm not a huge fan of all the bash-based testing bits here, and I think things are going to get pretty awkward real quick trying to develop the two sides independently